File: contest-3417.py
n = int(input())
for _ in range(n):
    i = int(input())
    find = False
    k = 0
    try:
        k = i - (9 * len(str(i)))
    except:
        k = 1
    
    for j in range(k, i):
        if j + sum((int(x) for x in str(j))) == i:
            print("Yes")
            find = True
            break
    if not find:
        print("No")
            

File: contest-3417.cpp
#include <iostream>

using namespace std;

bool f(int x)
{
    int k;

    if (x==0)
    {
        return true;
    }

    for (int i = 1; i <= 54; ++i)
    {
        int sum=0;
        k = x - i;
        for (k; k > 0; k /= 10)
        {
            sum += k % 10;
        }

        if (i==sum)
        {
            return true;
        }
    }

    return false;
}

int main ()
{
    int n, x;
    cin >> n;

    while (n--)
    {
        cin >> x;
        cout << (f(x) ? "Yes" : "No") << endl;
    }

    return 0;
}
